src/main/scala/chisel3/util/experimental/BulkConnect.scala

```diff
@@ -0,0 +1,89 @@
+package chisel3.util.experimental
+
```

Comment:
there was a *massive* comment on this file or a related one in the rocket-chip version explaining these operators. As someone who has repeatedly referred to that comment, I would want it upstreamed as well